overlord/configstate/configcore/timezone.go

```diff
@@ -34,6 +36,8 @@ import (
 func init() {
 	// add supported configuration of this module
 	supportedConfigurations["core.system.timezone"] = true
+	// and register it as a virtual config
+	config.RegisterVirtualConfig("core", "system.timezone", getTimezoneFromSystemVC)
```

Comment:
this is not related to the PR, but as we discussed in a the previous PR, virtual is maybe not the best term, it's about the implementation more than the use case.  Maybe s/Vitual/External/ in a follow up?

Reply:
Other options (assuming I got the intention right): dynamic, runtime.

Reply:
I added a XXX and will do it in a followup